package.json

```diff
@@ -2,9 +2,11 @@
   "name": "country-state-city",
   "version": "0.1.2",
   "description": "Library for fetching Country, its States and Cities",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "types": "index.d.ts",
```

Comment:
I haven't tried it locally due to lack of time, but are you sure your type def are not in the same folder as your complied code?

Reply:
forgot to change... it's in `dist/`